LINUX/veth_netmap.h

```diff
@@ -81,6 +81,7 @@ veth_netmap_dtor(struct netmap_adapter *na)
 		(struct netmap_veth_adapter *)na;
 	if (vna->peer_ref) {
 		vna->peer_ref = 0;
+		vna->peer->peer = NULL;
```

Comment:
do we need to reset ```vna->peer``` too?

Reply:
No - it is not necessary to set  vna->peer->peer for fixing bug, but I think it is better to remove reference to destructed peer.

Reply:
OK. I have reproduced the bug and confirmed that your patch fixes it. Thanks!